test-app/src/main/ml-schemas/tde/musician.tdex

```diff
@@ -0,0 +1,26 @@
+<template xmlns="http://marklogic.com/xdmp/tde">
```

Comment:
A couple things:

1. Make this a JSON file and use ".json" as the suffix. This is solely to avoid a Python user looking at our tests and seeing XML and thinking - yuck, I only like JSON and dictionaries, I'm going to be unhappy if I have to use XML for anything.
2. Change "opticUnitTest" to something simple like "test". (I don't like "opticUnitTest" in the context of the Java Client tests either because a) it's not used for "unit" tests, and b) it's too long to type).